tests/unit/tasks/server/middleware/tests-server-test.js

```diff
@@ -37,13 +31,17 @@ describe('TestServerAddon', function () {
           watcher: Promise.resolve()
         },
         finally: function() {
-          expect(nextWasCalled).to.true;
-          done();
+          try {
```

Comment:
there might be a better pattern for this, but I couldn't think of one while doing those commits...